src/AppBundle/EventListener/RedirectToPreferredLocaleListener.php

```diff
@@ -81,6 +81,10 @@ public function onKernelRequest(GetResponseEvent $event)
         if (!$event->isMasterRequest() || '/' !== $request->getPathInfo()) {
             return;
         }
+        // Ignore requests from referrers with the same HTTP host
```

Comment:
@bocharsky-bw, I think it's a bad comment, it doesn't explain why we should ignore a request. I would write a comment like this "We must handle only direct requests, because ...."